library/X509/DataTable.php

```diff
@@ -84,13 +84,16 @@ protected function renderRow($row)
         $cells = [];
 
         foreach ($this->columns as $key => $column) {
-            if (! is_int($key) && array_key_exists($key, $row)) {
```

Comment:
Since `array_key_exists` is different from `isset`, are you sure this will work as before?